packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -338,9 +358,7 @@ class CupertinoPageRoute<T> extends PageRoute<T> with CupertinoRouteTransitionMi
     super.settings,
     this.maintainState = true,
     super.fullscreenDialog,
-  }) : assert(builder != null),
-       assert(maintainState != null),
-       assert(fullscreenDialog != null) {
```

Comment:
Figured I might as well delete these on files that I'm touching to make the code cleaner (all of the assertion deletions are for null-checks on non-nullable parameters). I'm just assuming that there's nothing wrong with that, but lemme know if you'd like me to put these back to make the diff easier to read through